koku/api/settings/test/tests_views.py

```diff
@@ -43,159 +42,48 @@ def post_settings(self, body):
-            {"handler": AzureTagQueryHandler, "view": AzureTagView, "name": "azure"},
-            {"handler": GCPTagQueryHandler, "view": GCPTagView, "name": "gcp"},
+            {"handler": OCPTagQueryHandler, "view": OCPTagView, "name": "openshift", "label": "OpenShift labels"},
+            {"handler": AWSTagQueryHandler, "view": AWSTagView, "name": "aws", "label": "Amazon Web Services tags"},
+            {"handler": AzureTagQueryHandler, "view": AzureTagView, "name": "azure", "label": "Azure tags"},
+            {"handler": GCPTagQueryHandler, "view": GCPTagView, "name": "gcp", "label": "Google Cloud Platform tags"},
         ]
-        for test in test_matrix:
-            response = self.get_settings()
-            self.assertEqual(response.status_code, status.HTTP_200_OK)
-
-            duallist = self.get_duallist_from_response(response, test.get("name"))
-            all_keys = duallist.get("options")
-            self.assertIsNotNone(all_keys)
-            all_key_values = [key_obj.get("value") for key_obj in all_keys]
-            url = (
-                "?filter[time_scope_units]=month&filter[time_scope_value]=-1"
-                "&filter[resolution]=monthly&key_only=True&filter[enabled]=True"
-            )
-            query_params = self.mocked_query_params(url, test.get("view"))
-            handler = test.get("handler")(query_params)
-            query_output = handler.execute_query()
-            tag = query_output.get("data")[0]
-            self.assertIn(tag, all_key_values)
-
-    def test_post_settings_ocp_tag_enabled(self):
-        """Test setting OCP tags as enabled."""
-        test_matrix = [
-            {"handler": OCPTagQueryHandler, "view": OCPTagView, "name": "openshift"},
-            {"handler": AWSTagQueryHandler, "view": AWSTagView, "name": "aws"},
-            {"handler": AzureTagQueryHandler, "view": AzureTagView, "name": "azure"},
-        ]
-        for test in test_matrix:
-
-            url = (
-                "?filter[time_scope_units]=month&filter[time_scope_value]=-1"
-                "&filter[resolution]=monthly&key_only=True&filter[enabled]=True"
-            )
-            query_params = self.mocked_query_params(url, test.get("view"))
-            handler = test.get("handler")(query_params)
-            query_output = handler.execute_query()
-            tag = query_output.get("data")[0]
-
-            body = {"api": {"settings": {"tag-management": {test.get("name"): {"enabled": [tag]}}}}}
-            response = self.post_settings(body)
-            self.assertEqual(response.status_code, status.HTTP_200_OK)
-
-            response = self.get_settings()
-            self.assertEqual(response.status_code, status.HTTP_200_OK)
-            duallist = self.get_duallist_from_response(response, test.get("name"))
-            enabled = duallist.get("initialValue")
-
-            self.assertIn(tag, enabled)
-            tag = query_output.get("data")[1]
```

Comment:
Was the test code remove because it no longer applies?
Is there another test we can replace this with?

Reply:
The test code that was in place no longer applies and needs to be fully reworked. The settings page is now more then just the enable tabs object. I have added a @TODO for this work

Reply:
Can you please create a JIRA issue?

Reply:
Added additional unit test coverage